pythonmods/vlcplayer/__init__.py

```diff
@@ -37,10 +38,22 @@ def __init__(self):
         self.sayplayer = self.instance.media_player_new()
         self.list_player =  self.instance.media_list_player_new()
         self.list_player.set_media_player(self.player)
+        self.sc_api = SoundcloudAPI()
 
     def playytb(self, vid):
         self.play(vid2youtubeMRL(vid))
 
+    def playscloud(self, identifier):
+        #Url of the format: https://soundcloud.com/aries_ix/sayonara
+        url = "https://souncloud.com" + identifier
+        track = self.sc_api.resolve(url)
+        assert type(track) is Track
```

Comment:
Assertions are bad, since when they fail, it would tear down the susi_linux completely. Please to exception catching and proper error handling (simple logging is enough I guess)

Reply:
Okay @norbusan , i'll wrap it in a try catch block